Server/src/main.py

```
@@ -332,82 +342,176 @@ async def health_http(_: Request) -> JSONResponse:
+
+                        service = CustomToolService.get_instance()
+                        result = await service.execute_tool(
+                            project_id, tool_name, unity_instance_hint, tool_params
+                        )
+                        return JSONResponse(result.model_dump())
+
+                # Send command to Unity
+                result = await PluginHub.send_command(session_id, command_type, params)
+                return JSONResponse(result)
+
+            except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)